src/register/data/reducers.js

```diff
@@ -85,19 +91,30 @@ const reducer = (state = defaultState, action = {}) => {
         registrationError: { ...registrationErrorTemp },
       };
     }
+    case REGISTER_FORM_VALIDATIONS.BEGIN: {
+      return {
+        ...state,
+        validationsSubmitState: action.payload?.isValidatingMultiStepRegistrationPage
```

Comment:
we want to show spinner in button while fetching validations on step 1 and step 2. SO we need this state to know if validations are being fetched.

Reply:
as i explained in my other comments, we have two use cases for validations and I want this state to be set just only for the cta click flow. therefore I need this condition to differentiate between two flows.

otherwise there will be spinner on CTA even if the validations are being fetched on focus out of fields

Reply:
Ok, Got it, and thank you for explaining the use case.